src/world/partners.c

```diff
@@ -0,0 +1,216 @@
+#include "common.h"
+#include "ld_addrs.h"
+#include "partners.h"
+
+#include "partner/goombario.h"
+#include "sprite/npc/world_goombario.h"
+
+#include "partner/kooper.h"
+#include "sprite/npc/world_kooper.h"
+
+#include "sprite/npc/world_bombette.h"
+
+#include "sprite/npc/world_parakarry.h"
+
+#include "partner/goompa.h"
+#include "sprite/npc/goompa.h"
+
+#include "sprite/npc/world_watt.h"
+
+#include "sprite/npc/world_sushie.h"
+
+#include "sprite/npc/world_lakilester.h"
+
+#include "sprite/npc/world_bow.h"
+
+#include "partner/goombaria.h"
+#include "sprite/npc/goombaria.h"
+
+#include "partner/twink.h"
+#include "sprite/npc/twink.h"
+
+s32 world_partner_can_use_ability_default(Npc* partner);
+s32 world_partner_can_player_pause_default(Npc* partner);
+
+static s32 _pad[] = { 0x00, 0x00, 0x00 };
+
+WorldPartner gWorldPartners[] = {
+    // XXX: it's possible that's there's a "none" entry here to match up with enum PartnerID
```

Comment:
There's almost definitely a none entry. At least I found a loop that goes through partners and starts at 1. Maybe unrelated actually.

Reply:
Yeah, this pops up a bunch

Reply:
It'd be nice to figure out what RAM address this data is at, then we'd know